tools/measure_coverage.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+# Copyright 2017-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ "$#" -gt "0" ] && ( [ "$1" == "-h" ] || [ "$1" == "--help" ] ); then
+    echo "Measure JavaScript and C coverage and create a html report"
+    echo "out of the results"
+    echo ""
+    echo "Usage: $0 [NODE_MODULES_DIR]"
+    echo ""
+    echo "Optional Arguments:"
+    echo "  NODE_MODULES_DIR    Specifies the node_module directory, where"
+    echo "                      the nodejs dependencies are installed."
+    echo ""
+    echo "The created html reports can be found in the 'coverage' directory,"
+    echo "which will be created in the IoT.js project source dir. The C and"
+    echo "JavaScript coverage reports are in the 'c' and 'js' subdirectories"
+    echo "respectively. The reports can be viewed by opening the 'index.html'"
+    echo "file in a web browser of your choice."
+    echo ""
+    echo "Running the script will require some additional dependencies:"
+    echo "    sudo apt install npm lcov nodejs gcc-multilib"
+    echo "    npm install --save-dev babel-cli nyc babel-plugin-istanbul"
+    echo ""
+    exit 0
+fi
+
+tools_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+iotjs_root=$(realpath "$tools_dir/..")
+
+dpkg -l npm >> /dev/null 2>&1 && \
+dpkg -l lcov >> /dev/null 2>&1 && \
+dpkg -l nodejs >> /dev/null 2>&1 && \
+dpkg -l gcc-multilib >> /dev/null 2>&1
+
+if [ "$?" -ne "0" ]; then
+    echo "Please install all required dependencies"
+    echo ""
+    echo "    sudo apt install npm lcov nodejs gcc-multilib"
+    echo "    npm install --save-dev babel-cli nyc babel-plugin-istanbul"
```

Comment:
I tried to run only ``sudo apt install npm lcov nodejs gcc-multilib`` command and run this script again, I got the error message.

```bash
nova@nova-VirtualBox:~/iotjs/coverage$ ./tools/measure_coverage.sh 
./tools/measure_coverage.sh: line 41: realpath: command not found
./tools/measure_coverage.sh: line 56: realpath: command not found
Could not find the node_modules directory, please specify it
as an argument.
```

It looks better to show ``npm install --save-dev babel-cli nyc babel-plugin-istanbul`` in this case.

Also, does this script have dependency ``realpath``?